docs/ai/welcome.md

```diff
@@ -38,7 +40,7 @@ You can confirm installation by running `subql-ai --help`.
 
 ## Create a new App
 
-You can initialise a new app using `subql-ai init`. It will ask you to provide a name and a Ollama model to use.
+You can initialise a new app using `subql-ai init`. It will ask you to provide a name and a LLM model to use.
```

Comment:
So what happens when you init and select an OpenAI model? does it allow you to select between GTP4, GPT4-o, etc?

Does it prompt for an API key or is this automated?

Reply:
Yes, you can select chat GPT models.

It won't prompt for an API key, that won't be stored in the project as its sensitive information